doc/source/structures/celestial_bodies/body.rst

```diff
@@ -67,6 +67,9 @@ All of the main celestial bodies in the game are reserved variable names. The fo
     :attr:`NAME`                     :ref:`string <string>`
     :attr:`DESCRIPTION`              :ref:`string <string>`
     :attr:`MASS`                     :ref:`scalar <scalar>` (kg)
+    :attr:`OCEAN`                    :ref:`boolean <boolean>`
```

Comment:
I think this should probably be named with the word "HAS" in front of it, to make it consistent with many of the other similar boolean suffixes like this (i.e. call it "HASOCEAN").

Thanks for the contribution, by the way!

Reply:
You're welcome. Fixed